pkg/vault/test/authorizer/TimelockAuthorizer.test.ts

```diff
@@ -1585,6 +1774,18 @@ describe('TimelockAuthorizer', () => {
                   await authorizer.execute(id);
                   await expect(authorizer.execute(id)).to.be.revertedWith('ACTION_ALREADY_EXECUTED');
                 });
+
+                it('receives canceler status', async () => {
+                  const id = await schedule();
+
+                  expect(await authorizer.isCanceler(id, grantee)).to.be.true;
+                });
+
+                it('can cancel the action immediately', async () => {
+                  const id = await schedule();
+                  // should not revert
+                  await expect(authorizer.cancel(id, { from: grantee })).not.to.reverted;
```

Comment:
You can test for the `ExecutionCancelled` event here, which is slightly more robust